testing/tests/ui/concat.rs

```diff
@@ -0,0 +1,46 @@
+use rinja::Template;
```

Comment:
Can you also add these two cases:

```
~ a
a ~
```

please?

Reply:
Added.